microservices-orders/start.sh

```diff
@@ -7,8 +7,8 @@ check_env || exit 1
 check_mvn || exit 1
 check_jot || exit 1
 check_netstat || exit 1
-check_running_elasticsearch 5.6.16 || exit 1
-check_running_kibana || exit 1
+#check_running_elasticsearch 5.6.16 || exit 1
+#check_running_kibana || exit 1
```

Comment:
My guess is this line was commented out for quick validation, but please uncomment before PR merge.